pytorch_lightning/core/memory.py

```diff
@@ -213,6 +213,56 @@ def out_sizes(self) -> List:
     def param_nums(self) -> List[int]:
         return [layer.num_parameters for layer in self._layer_summary.values()]
 
+    @property
+    def total_out_params(self) -> int:
+        _total_out_params = 0
+
+        def _get_out_size_params(out_sizes):
+            nonlocal _total_out_params
+            if not any(isinstance(i, list) for i in out_sizes):
+                try:
+                    # try to find prod, i.e check for unknown sizes.
+                    _total_out_params += np.prod(out_sizes)
+                except:
+                    # do nothing if could not find product.
+                    pass
+            else:
+                _ = [_get_out_size_params(out_size) for out_size in out_sizes if isinstance(out_size, list)]
+
+        _get_out_size_params(self.out_sizes)
+
+        return _total_out_params
+
+    @property
+    def total_params(self) -> int:
+        return sum(self.param_nums)
+
+    @property
+    def model_size(self, input_size=None) -> float:
```

Comment:
If you define this as a property, you can never pass an argument here...

Reply:
@justusschock  yeah was removing this but missed in commit